components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -283,6 +282,9 @@ class EventsStorageEngineTest {
 
             Assert.assertTrue(click.testHasValue())
 
+            // Wait for the ping to be written to disk.
+            Glean.pingStorageEngine.testWait()
```

Comment:
Would it be less error-prone to put this inside of `triggerWorkManager()`?  (i.e., even if it's slower, I'd be in favor of making it easier to avoid async pitfalls in tests).

Reply:
I think it would. Do you mind if I file a bug and take that as a future follow-up?

Reply:
Sure.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1540068